src/shared/components/oncoprint/DeltaUtils.ts

```diff
@@ -1275,11 +1277,12 @@ function transitionClinicalTrack(
             important_ids: nextSpec.altered_uids, // only show clinical track legends for altered cases. e.g. if only altered cases are female for the SEX clinical track, then only show 'female' in the SEX legend
             //track_info: "\u23f3",
             sortCmpFn: getClinicalTrackSortComparator(nextSpec),
-            init_sort_direction: 0 as 0,
+            init_sort_direction: getClinicalTrackInitialSortDirection(nextSpec),
             target_group: CLINICAL_TRACK_GROUP_INDEX,
             onSortDirectionChange: nextProps.onTrackSortDirectionChange,
             custom_track_options: nextSpec.custom_options,
             track_can_show_gaps: nextSpec.datatype === 'string',
+            init_show_gaps: getClinicalTrackInitialShowGapsConfig(nextSpec),
```

Comment:
Mental note: apparently, _show_gaps_ is not always possible (see `track_can_show_gaps` prop). We need to make sure that we only try to apply gaps when this is possible.

Reply:
I checked this, they're not added when not possible.